### PR TITLE
Fix rendering for default/example values of 0.

### LINKED ISSUE #5573: default zero not allowed in parameters
"pageNumber": {
			"name": "pageNumber",
			"in": "query",
			"type": "integer",
			"required": true,
			"description": "Page number is location of page",
			"default": 0
		},

image

### PR DESCRIPTION
**Pull request by danxmoran**
### Description
Instead of discarding all default/example values which are false-y, only discard those that are `undefined`.

### Motivation and Context
Fixes #5573.

### How Has This Been Tested?
The included mocha test failed before I changed the main code, and passes after. I haven't done any additional testing (I tried playing with selenium but couldn't get it to work on my machine).

### My PR contains... 
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [x] My changes can and should be tested by unit and/or integration tests.
- [x] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/core/components/parameter-row.jsx [hunk 1]
```diff
@@ -119,17 +119,21 @@ export default class ParameterRow extends Component {
       //// Find an initial value
 
       if (specSelectors.isSwagger2()) {
-        initialValue = paramWithMeta.get("x-example")
-          || paramWithMeta.getIn(["schema", "example"])
-          || (schema && schema.getIn(["default"]))
+        initialValue = [
```
Comment (tim-lai) [inline src/core/components/parameter-row.jsx:122]: I think it would be better NOT to use array and array.find() for `initialValue`. Seems like it would be simpler to add a secondary validation check to each operation as they get called.

**Comment by tim-lai** [comment]
@danxmoran Thanks for the PR! I tried rebasing with master, but now it fails Cypress tests in `features/schema-form.js`. I think it is because `initialValue` is now different than expected from the test. I noted in comment about the use of array. In terms of tests, I think the relevant additional  mocha tests besides `0` should include, `undefined`, and other `falsy` type values.

File: src/core/components/parameter-row.jsx (hunk 1 above)
Comment (tim-lai) [reply src/core/components/parameter-row.jsx:122]: update: this change is valid

**Comment by tim-lai** [comment]
superceded by #6454
